app/controllers/cars_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CarsController < ApplicationController
+  def new
+    @user = current_user
+  end
+
+  def create
+    if missing_params?
+      flash.now[:error] = 'Missing fields'
+      render cars_new_path
+    else
+      CarsFacade.new_car(new_car_params, current_user)
+      redirect_to "/car_monthly_mileages/new"
+    end
+  end
+
+  def missing_params?
+    new_car_params[:make] == "" || new_car_params[:model] == "" ||
+    new_car_params[:year] == "" || new_car_params[:mpg] == "" ||
+    new_car_params[:fuel_type] == "Select Fuel Type"
```

Comment:
This might be something we should handle at the poro level, since it seems the be input validation.